README.md

```diff
@@ -104,3 +104,26 @@ You can setup the same configuration but this time using [Enterprise Shield](htt
 0. Open a second browser tab at [https://kaazing.example.com/demo/](https://kaazing.example.com/demo/) and log in with username **alice** and password **alice**.
 
 0. From the **alice** tab, you can now call bob.
+
+
+##### Running High Availability (Optional)
+
+You can setup the same configuration in a high-availability configuration. High-availability is created at the DNS level. 

```

Comment:
@DoruM can we change this so there are 2 coturns in the back.  Also a diagram would be great way of depicting this